### ISSUE TITLE
[BUG] iOS screen reader is not supported

### ISSUE THREAD

**Issue by IsaMorphic** [AvaloniaUI/Avalonia#20125]
### Describe the bug

Accessibility is not a feature, it is core functionality. Currently, Avalonia on iOS does not support accessibility tools like iOS VoiceOver. This is a merge request for #18016. 

### To Reproduce

Turn on iOS VoiceOver while using any Avalonia app and observe that nothing gets read aloud. 

### Expected behavior

iOS VoiceOver should function as expected by reading aloud all on-screen content and controls, with proper labelling for buttons, images, selection boxes, etc.

### Avalonia version

Nightly

### OS

iOS

### Additional context

This is a merge request. The functionality in question is completed in the PR linked above. I urge maintainers to prioritize merging this critical functionality so that users with disabilities can enjoy Avalonia, too :)

**Comment by IsaMorphic** [comment]
@grokys @maxkatz6 We could really use a review on this critical patch ❤️

**Comment by LyleRolleman-plst** [comment]
Will add my voice to this, lack of accessibility is a major blocker for enterprise and government applications and really should be prioritized
